drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -251,6 +251,9 @@ K_MEM_POOL_DEFINE(ep_buf_pool, EP_BUF_POOL_BLOCK_MIN_SZ,
  */
 struct nrf_usbd_ctx {
 	usb_dc_status_callback status_cb;
+	struct onoff_client hfxo_cli;
+	struct onoff_manager *hfxo_mgr;
+	atomic_t clk_single_req;
```

Comment:
Please provide some description for those as it is done for other fileds in this struct. Also Could You please rename `clk_single_req` back to `clock_requested`? I assume it has same purpose but older name says more to me.